test/unit/virtual_include/inc/zeroDiv.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 Ericsson AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ZERODIV_H
+#define ZERODIV_H
+int warnMe(){
+    return 1/0;
+}
```

Comment:
Tip: If you don't want to have compiler warnings (even plain compiler warnings can catch this), add the tinyiest bit of control flow. 
```suggestion
int warnMe(int i){
    if (i == 0)
        return 1/i;
}
```